sklearn/model_selection/tests/test_validation.py

```diff
@@ -1985,15 +1985,3 @@ def _more_tags(self):
            "Set the estimator tags of your estimator instead")
     with pytest.warns(FutureWarning, match=msg):
         cross_validate(svm, linear_kernel, y, cv=2)
-
-    # the _pairwise attribute is present and set to True while the pairwise
```

Comment:
@NicolasHug by not being permissive (getting default with _get_tags), we need to remove this test. What do you think about this.

Reply:
We are not sure that this case is actually possible in practice.

Reply:
I haven't followed the introduction of the `pairwise` tag but since the test assumes that the tag doesn't exist and since we're telling 3rd parties that all tags should exist, I'd say it makes sense to remove the test